### PR TITLE
📖 stories(docs): add Form stories and docs

### PR DESCRIPTION
**Pull request by marcomauricio1**
**Why these changes?**

- adding documentation and stories for `Input`, `InputGroup`, `FormGroup`, and `Validation`

**How do we test?**

`pnpm start`

**Screenshots**
## FormGroup
Screenshot 2023-12-14 at 1 06 26 PM
## InputGroup
Screenshot 2023-12-14 at 1 06 54 PM
Screenshot 2023-12-14 at 1 07 03 PM
## Validation
Screenshot 2023-12-14 at 1 07 52 PM
## Input
Screenshot 2023-12-14 at 1 08 09 PM
Screenshot 2023-12-14 at 1 08 18 PM
Screenshot 2023-12-14 at 1 08 29 PM
Screenshot 2023-12-14 at 1 08 38 PM

### CODE REVIEW

File: src/FormGroup/FormGroup.svelte [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
+
+  /**
+   * Indicates whether the FormGroup is part of a row layout.
+   * @type {boolean}
+   */
   export let row = false;
+
+  /**
+   * A custom tag for the FormGroup
+   * @type {?string}
```
Comment (dysfunc) [inline src/FormGroup/FormGroup.svelte:52]:
```suggestion
   * @type {string}
```

File: src/Input/Input.stories.svelte [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+</script>
+
+<Template let:args>
+  <Input {...args} />
+</Template>
+
+<Story name="Basic" />
+
+<Story name="Text">
+  <div class="input-example">
```
Comment (dysfunc) [inline src/Input/Input.stories.svelte:180]: I would restrict the width of all the examples to something like 300px or 400px. it looks ugly spanning the entire length

Comment (dysfunc) [reply src/Input/Input.stories.svelte:180]: this include inputgroup and other stories

**Comment by marcomauricio1** [comment]
Screenshot 2023-12-14 at 3 02 24 PM

**Comment by marcomauricio1** [comment]
400px width applied to all stories

**Review by dysfunc — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**